ControllerCommon/Utils/InputUtils.cs

```diff
@@ -59,10 +59,11 @@ public enum OverlayModelMode
     {
         OEM = 0,
         Virtual = 1,
-        ZDOPlus = 2,

```

Comment:
Not a big deal, as I also like the idea of having XboxOne on top of the list but we should in the future try not to change the index order because it'll affect users and their previous settings value.